dataplane/flags/flags.go

```diff
@@ -873,6 +873,13 @@ var (
 			EnvVar: "CB_WORKSPACE",
 		},
 	}
+	FlWithSdxOptional = BoolFlag{
```

Comment:
I am not sure about the naming here. As far as I know Data Lake is the preferred term now, however the subcommand to manipulate data lakes still called sdx. I am not sure which one is better: to be consistent or to be accurate. Could you discuss this with @doktoric?

Reply:
We don't have an official name for this. We use sdx and datalake as well